src/tabs/tab-header-bar.tsx

```diff
@@ -64,6 +71,20 @@ export function TabHeaderBar({
   const [horizontalOverflow, setHorizontalOverflow] = useState(false);
   const [inlineStartOverflow, setInlineStartOverflow] = useState(false);
   const [inlineEndOverflow, setInlineEndOverflow] = useState(false);
+  const [previousActiveTabId, setPreviousActiveTabId] = useState<string | undefined>(activeTabId);
+  const hasActionOrDismissible = tabs.some(tab => tab.action || tab.dismissible);
+  const tabsWithActionsAriaRoleDescription = 'Tabs with Actions';
```

Comment:
There shouldn't be a default string here. Instead it should be using the prop from `i18nStrings`